File: chapter2/sum.py
number_1 = int(input("enter first number: "))
number_2 = int(input("enter second number: "))
number_3 = int(input("enter third number: "))

sum = number_1 + number_2 + number_3
average = sum / 3
product = number_1 * number_2 * number_3
smallest = min(number_1, number_2, number_3)
largest = max(number_1, number_2, number_3)

print(f"sum: {sum}\naverage: {average}\nproduct: {product}\nsmallest number: {smallest}\nlargest number: {largest}")

File: functionSnacks/Sum.java
public class Sum {
	public static void main(String... args) {
		int[] number = {4, 7, 23, 89, 1, 22};

		System.out.println(largest(number));
		System.out.println(smallest(number));
	}

	public static int largest(int[] array) {
		int largest = array[0];

		for (int count = 0; count < array.length; count++) {
			if (array[count] > largest) {
				largest = array[count];
			}
		}
		
		return largest;
	}

	public static int smallest(int[] array) {
		int smallest = array[0];

		for (int count = 0; count < array.length; count++) {
			if (array[count] < smallest) {
				smallest = array[count];
			}
		}
		
		return smallest;
	}
}